lib/sql.ml

```diff
@@ -242,6 +242,7 @@ type ctor =
 and var =
 | Single of param
 | SingleIn of param
+| ChoiceIn of param_id * bool * var list
```

Comment:
```suggestion
| ChoiceIn of { param : param_id; default : bool; vars : var list }
```

Reply:
I wonder f ChoiceIn can be somehow avoided with new variant(s) of `ctor` - ie one representing "ensure parameter is non empty list" case. Not sure if it is worth it (supposedly it will simplify handling of params but will require more attention at time of destructing user value - this ctor cannot be mixed with other ctors)